### ISSUE TITLE
Entypo Social Extension Fonts not loading properly across browsers/OS

### ISSUE THREAD

**Issue by bnjmn** [danielbruce/entypo#10]
So far I've noticed the following
- Windows 7
  - Chrome: Does not load social fonts at all
  - Firefox: Social fonts load, but all entypo characters seem to be at a lower resolution (this may be unavoidable)
- Ubuntu 12.04
  - Both Chromium and Firefox seem to perform consistently

**Comment by danielbruce** [comment]
> So far I've noticed the following
> 
> Windows 7
> 
> Chrome: Does not load social fonts at all
> Firefox: Social fonts load, but all entypo characters seem to be at a lower resolution (this may be unavoidable)
> Ubuntu 12.04
> 
> Both Chromium and Firefox seem to perform consistently

If you regenerate the @font-face fonts from the OpenType will that help?

–

Daniel Bruce  |  Creative
A good idea is to follow me on Twitter and Dribbble.

**Comment by bnjmn** [comment]
I'm using the same implementation as on the entypo.com site (embedded opentype). I've noticed the same results on the site as on my own, e.g., the social fonts are not displayed on entypo.com with Chome (W7) either.

**Comment by kfalkiewicz** [comment]
Regenerated version from www.fontsquirrel.com isn't working either (problem in Chrome and Opera on W7).
I tried and used version available from: www.icomoon.io/app with changed mapping from Private Area unicode to normal characters like A,B,C..... etc. and it works. I think that there is problem with mapping unicode characters and some browsers just don't render them. (font is loaded properly, and programs see this glyphs inside font).

**Comment by madelinecr** [comment]
+1, impacted by this issue as well.

**Comment by danielsokolowski** [comment]
Impacted as well - https://github.com/styleguide/css/7.0 appears to be using private Unicode area as well with no issues.

**Comment by kfalkiewicz** [comment]
@danielsokolowski your link isn't perfect also. On Mac OS X Chrome doesn't see 2 icons addressed to : UTF+f288 and UTF+f289. Problem is in some browsers, becouse their don't recognize all private Unicode area, so better to use letters for it.

**Comment by danielsokolowski** [comment]
Yes I see that, thank you - learning something new all the time.

**Comment by jackmcdade** [comment]
any fix for this coming? They're completely dead in numerous browsers.

**Comment by bryan-farris** [comment]
@jackmcdade having issues here too.

**Comment by drush** [comment]
+1 @danielbruce Any idea of the root cause on this issue?  Need an assistance getting it resolved?

**Comment by ariabov** [comment]
+1, getting this fixed would really help

**Comment by wo0** [comment]
Your icon set is super, super helpful!  But, +1 for the Chrome issue too :).

**Comment by Pixelous** [comment]
Still no solution?

**Comment by irvinebroque** [comment]
+1 - I believe this is related: https://code.google.com/p/chromium/issues/detail?id=122465

**Comment by nolastan** [comment]
For me the problem was `text-rendering: optimizeLegibility;` applied site-wide. Setting `text-rendering: auto;` for the icons fixed rendering issues in Windows Chrome.

**Comment by SergioCrisostomo** [comment]
Hi @danielbruce , is there a solution for this issue?

**Comment by danielbruce** [comment]
Sorry for this late reply. The problem has to do with the unicode positions of the characters. Download a new version of the typeface from fontello.com and that will solve all problems. I will aim to use that engine for the next version of Entypo.

**Comment by eeeeeeeeeve** [comment]
Looks like this was fixed for the standard set, but not for the social extension:

![screenshot 2014-02-20 16 58 27](https://f.cloud.github.com/assets/790381/2216580/2f5d5b54-9a0d-11e3-824d-56c97c4932bb.png)

**Comment by sbornia** [comment]
Yep, social icons not working on Chrome - Windows...

**Comment by kflorence** [comment]
+1 -- social font was not updated.

**Comment by montogeek** [comment]
2 months later is the problem still exists :(

**Comment by kflorence** [comment]
I've switched to using FontAwesome because their font sets work :)

**Comment by montogeek** [comment]
Yes, the last updated on the Entypo repo is 2 years ago :/

Fernando Montoya
montogeek
http://www.bytedesk.com http://montogeek.blogspot.com
01001101 01101111 01101110 01110100 01101111 01000111 01100101 01100101
01101011 10011001

Recibo emails VSRE. Aprende qué es en http://vsre.info/

**Comment by CourtFantinato** [comment]
+1 for Entypo Social extension still not working. Looks like I'll need to use something else...

**Comment by jackmcdade** [comment]
I think it's time to let it go. Unwatching and moving on.

**Comment by mandynicole** [comment]
Using <strong><a href="http://www.fontello.com">Fontello</a></strong> to cherry-pick which icons you want (and from whatever library you want) still works and is probably what most need anyway, not to mention I love the service and highly encourage everyone use it. This is what I did, and the social icons are working fine in Windows on my personal site.

@danielbruce Is there an ETA for 3.0? Hate to see people drifting away from the best free icon font set out there due to this.

**Comment by chriswalkr** [comment]
+1 this is still an issue. Also doesn't work on your own website http://www.entypo.com/characters/ (In Windows 7 and Chrome)

**Comment by conlanpatrek** [comment]
Not that this is helpful...

But I believe the issue is a problem with Chrome displaying certain unicode characters in Windows. If you remap missing glyphs to unicode alphanumeric, you can solve this issue and get them to display. You'll just have to remember your new mappings.
